app/src/main/java/org/sopt/sample/data/remote/ResponseUserList.kt

```diff
@@ -0,0 +1,28 @@
+package org.sopt.sample.data.remote
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseUserList(
```

Comment:
```suggestion
data class ResponseUser(
```
해당 data class가 List에 담기기 직전이기 때문에 `ResponseUserList` 라는 이름은 다소 어색하단 느낌이 듭니다..!
`ReponseUser` 가 더 좋을 것 같습니다..!